drafts/erc20-rwa-disclosure-uri.md

```diff
@@ -0,0 +1,38 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 8104
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.